css/rules.css

```diff
@@ -0,0 +1,27 @@
+.main__container {
+  width: 100%;
+  height: 100vh;
+}
+
+.container__rules {
+  width: 80rem;
+  height: 60rem;
+  /* background-color: var(--primary-shade); */
+  border: 0.1rem solid var(--darker);
+  border-radius: 2rem;
+}
+
+.container__heading {
+  width: 90%;
+  font-family: var(--font-heading);
+  font-size: 4rem;
+  border-bottom: 0.1rem solid var(--darker);
+}
+
+.rules_points {
+  width: 90%;
+}
+
+.rule-text {
+  font-size: 3rem;
```

Comment:
Is your class "rule-text" a generic class? If not you can try making classes like fs-3 or txt-3 and use it whenever you need. Got this advice from the mentor.

Reply:
sure paarth but as of now i have not added font-size utilities in my component library when i will add i'll sure take this approach. thanks.